packages/sitecore-jss/src/native-fetcher.test.ts

```diff
@@ -7,12 +7,20 @@ import debug from './debug';
 
 use(spies);
 
-let fetchInput: URL | RequestInfo | undefined;
+let fetchInput: RequestInfo | undefined;
 let fetchInit: RequestInit | undefined;
 
-const mockFetch = (status: number, response: unknown = {}, jsonError?: string) => {
+const mockFetch = (
+  status: number,
+  response: unknown = {},
+  {
+    jsonError,
+    textError,
+    responseType,
+  }: { jsonError?: string; textError?: string; responseType?: 'text' | 'json' } = {}
+) => {
   return (input: URL | RequestInfo, init?: RequestInit) => {
-    fetchInput = input;
+    fetchInput = input instanceof URL ? input.toString() : input;
```

Comment:
why is this change needed?

Reply:
aah i forgot to revert this.
changed the `fetchInput` type above.